torch/tensor.py

```diff
@@ -349,12 +349,14 @@ def stft(self, n_fft, hop_length=None, win_length=None, window=None,
     def resize(self, *sizes):
         warnings.warn("non-inplace resize is deprecated")
         from torch.autograd._functions import Resize
-        return Resize.apply(self, sizes)
+        new_tensor = Resize.apply(self, sizes)
+        return torch.Tensor._make_subclass(self.__class__, new_tensor.data, new_tensor.requires_grad)
```

Comment:
I'm not sure if it actually matters here (since these are deprecated methods on tensor), but the loss of autograd metadata here makes me raise my eyebrows. I haven't reviewed how `_make_subclass` works yet, but I don't see how you could possibly preserve ResizeBackwards in this case, because you have only passed along the detached tensor and whether or not it requires grad. But that's not all the information that a Variable may have (e.g., `grad_fn`!)

Reply:
Yeah, I think I detached by mistake again. I will fix this.